src/components/ListSelectionPage/ListSelectionPage.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { InjectAppServices } from '../../services/pure-di';
+import { Loading } from '../Loading/Loading';
+import { SubscriberListState } from '../../services/shopify-client';
+import { Promotional } from '../ContactPolicy/Promotional';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import { useFetch } from '../../hooks/useFetch';
+
+const listsPerPage = 20;
```

Comment:
Normally I declare the variables in Uppercase and outside the component, when their value is known before the execution of the app. Probably later, you want to standardize so that the tables to select lists work with this same value. It may be a good idea to raise the variable to a level where it can have a greater meaning and change its name.
I think in this article they talk a little about that https://www.codigofuente.org/variables-y-constantes-en-javascript/

Reply:
Nice article, thanks. It would be nice to share it with the team. The idea here is to follow the `camelCase` format that is used to declare global variables in utils: `export const successMessageDelay = 3000;`